Dockerfile

```diff
@@ -0,0 +1,17 @@
+# syntax=docker/dockerfile:1
+FROM maven:3.8.6-openjdk-11
+
+COPY . /datatools
+
+# Build jar
+WORKDIR /datatools
+RUN mvn package -DskipTests
+RUN cp target/dt*.jar ./datatools-server.jar
```

Comment:
I get this error locally (using `colima` on MacOS). Is it specific to me?
```
 => [3/6] WORKDIR /datatools                                                                                                                                                                    0.0s
 => [4/6] RUN mvn package -DskipTests                                                                                                                                                          57.2s
 => ERROR [5/6] RUN cp target/dt*.jar ./datatools-server.jar                                                                                                                                    0.5s
------
 > [5/6] RUN cp target/dt*.jar ./datatools-server.jar:
#13 0.440 cp: target './datatools-server.jar' is not a directory
------
failed to solve: rpc error: code = Unknown desc = executor failed running [/bin/sh -c cp target/dt*.jar ./datatools-server.jar]: exit code: 1
```

Reply:
Are you building the image on your own or via the docker-compose file?

Reply:
@binh-dam-ibigroup it wasn't your fault! Things are now fixed.

Reply:
I ran this:
```
colima start
 docker-compose -f docker-compose.yml up --abort-on-container-exit
```

Reply:
Yep that command looks excellent. If you run that with the latest commit things should work!

Reply:
I see this next:
```
mongo_1             | {"t":{"$date":"2022-09-22T14:45:51.347+00:00"},"s":"I",  "c":"INDEX",    "id":20345,   "ctx":"LogicalSessionCacheRefresh","msg":"Index build: done building", ...
datatools-server_1  | Error: Unable to access jarfile datatools-server.jar
datatools-server_1 exited with code 1
Aborting on container exit...
```

Reply:
Happy to have someone reviewing this! Have made the jar renaming a bit more resilient

Reply:
I'm still seeing the above error for some reason.

Reply:
Did you clear your docker cache? Try running `docker-compose down` and `docker system prune --all` and then running `docker-compose up` again

Reply:
Next error:
```
 => [5/7] RUN cp target/dt*.jar /datatools/                                                                                                                                                     1.2s
 => ERROR [6/7] RUN mv dt*.jar datatools-server.jar                                                                                                                                             0.7s
------
 > [6/7] RUN mv dt*.jar datatools-server.jar:
#14 0.649 mv: target 'datatools-server.jar' is not a directory
------
```

Reply:
I'm no longer able to replicate this. Something might be wrong with your docker filesystem? I've run `colima delete`, `colima start` and `docker-compose up` and am unable to replicate this (although was able to replicate the previous issues you noted)